### DISCUSSION TITLE
How to remove the version "V4.9.2" that shows up in the Bot status?

### DISCUSSION THREAD

**Discussion by priyam-raj** [Androz2091/AtlantaBot#461] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/26673536/117570163-b58fd100-b0e6-11eb-8c72-1bace16a18e7.png)

**Comment by karlof002** [comment, marked as answer]
Hey, you should go to the `ready.js` file and check the setInterval method, search for it and remove it, regardless

**Reply by Androz2091** [reply]
Yes, you should edit this part of the bot. Don't remove the setInterval, but only the + require("../package.json").version

**Reply by priyam-raj** [reply]
Thank you, it worked
